resources/i18n/it_IT/cura.po

```diff
@@ -4336,30 +3948,29 @@ msgctxt "@info:status Don't translate the XML tag <filename>!"
 msgid "Successfully imported material <filename>%1</filename>"
 msgstr "Materiale importato correttamente <filename>%1</filename>"
 
-#, python-brace-format
 msgctxt "@info:status"
 msgid "Successfully imported profile {0}."
 msgstr "Profilo {0} importato correttamente."
 
 msgctxt "@action:label"
 msgid "Suggested Material settings"
-msgstr ""
+msgstr "Impostazioni del materiale consigliate"
```

Comment:
even "Impostazioni consigliate del materiale" if "suggested" is related to "settings"